docs/docs/cmd/planner/task/task-add.md

```diff
@@ -49,9 +49,18 @@ m365 planner task add [options]
 `--assignedToUserNames [assignedToUserNames]`
 : The comma-separated UPNs of the assignees the task is assigned to. Specify either `assignedToUserIds` or `assignedToUserNames` but not both.
 
+`--assigneePriority [assigneePriority]`
+: Hint used to order items of this type in a list view
+
 `--description [description]`
 : Description of the task
 
+`--appliedCategories [appliedCategories]`
+: Comma-separated categories that should be added to the task
```

Comment:
In the command `task set`, we currently do it this way. However, it's a good idea to put more clarification on this. The added url is likely to cause some confusion. Mainly because the structure within the `plannerAppliedCategories` resource type is different than how we use it within the CLI. 

`plannerAppliedCategories` mentions the following json representation.

``` json
{
  "category1": true,
  "category3": true,
  "category5": true
}
```

Within the CLI, we pass it along as a comma-separated string. `category1,category3`

Reply:
In that case let's add the URL _and_ a short extra description of the values that we expect.